tests/components/application-connector/resources/charts/compass-runtime-agent-test/values.yaml

```diff
@@ -7,13 +7,11 @@ skipDirectorCertVerification: true
 serviceAccountName: "test-compass-runtime-agent"
 
 containerRegistry:
-  path: "eu.gcr.io/kyma-project"
-
+  path: "docker.io/koala7659"
```

Comment:
I think we should revert this change too

Reply:
Done